src/plane.ts

```diff
@@ -9,19 +9,104 @@
 // -- Remove boxMesh and coneMesh
 
 import * as THREE from 'three';
+import {option} from 'fp-ts';
+import * as defaultPlane from './default-plane.json';
+
+interface basicGeometry {
+  wingspan: number;
+  length: number;
+  chord: number;
+}
+
+enum toControls {
```

Comment:
I don't understand the meaning of "to" here. If "interrupts to controls (yaw|pitch|roll)" is a phrase that makes sense in aerospace, it'd be better for the "to" to go on "interruptsTo". The enum type itself lacks the context it's used in and may be reused in other contexts, so I think it's better to just call it the more generic Controls.

Reply:
I've already changed it, but I think that the way I was thinking of it was as a filter from a raw JSON to a type. Probably not the best way to think of it, but the schema does that for me for the most part anyways now.